### ISSUE TITLE
JOSS: Installation instructions are platform specific and incomplete

### ISSUE THREAD

**Issue by dfm** [KenethGarcia/ClassiPyGRB#1]
This code isn't Linux specific so it seems somewhat limiting to only include `apt-get` installation instructions for dependencies.

One more cross platform option would be conda/mamba. The steps that worked for me:

```bash
mamba create -n ClassiPyGRB -c conda-forge python=3.10 pytables=3.8.0 pillow
mamba activate ClassiPyGRB
pip install .
```

Note that the correct version of PyTables needs to be installed with conda/mamba if you don't have the appropriate hdf5 dev library, which is not currently listed as a dependency.

I'd also argue that it's probably unnecessary to pin all the dependency versions (rather than just bounding them). But if there's a good argument for pinning them like that, that should be clearly stated in the README.

ref: https://github.com/openjournals/joss-reviews/issues/5923

**Comment by KenethGarcia** [comment]
Thank you for your comment! We want to highlight the following points for this issue:

1. We bounded all the requirements for **ClassiPyGRB**. As you mentioned, there is no reason to pin any package. We ran all the tests and tutorials using the latest versions (up to Dec. 20th), and there are no issues to fix.
2. We updated the installation tutorial by adding conda/mamba installation instructions. We want to stress that we conducted several hours of testing and environment validation to ensure a seamless and reliable installation process. We remark also that in previously installed tkinter/pytables OS, the requirements creating the conda environment are unnecessary. 
3. As the [PyTables documentation states](https://www.pytables.org/usersguide/installation.html), the need for the `tables` requirement for the package was already established as a dependency for installations with pip (see the [TOML file](https://github.com/KenethGarcia/ClassiPyGRB/blob/master/pyproject.toml)). However, we mention to install PyTables in conda/mamba environments on the README.
